platform/branding-api/pom.xml

```diff
@@ -21,9 +21,8 @@
         <artifactId>platform</artifactId>
         <version>2.11.0-SNAPSHOT</version>
     </parent>
-    <groupId>ddf.distribution</groupId>
     <artifactId>branding-api</artifactId>
```

Comment:
I could change these artifact ids if folks would prefer that. I was just trying to make as few changes as possible.